appveyor.yml

```diff
@@ -13,9 +13,9 @@ environment:
   # Do not move these lines; they are referred to by README.md.
   # Versions of go that are explicitly supported by gonum.
   matrix:
-    - GOROOT: 'c:\go19'
     - GOROOT: 'c:\go110'
     - GOROOT: 'c:\go111'
+    - GOROOT: 'c:\go112'
```

Comment:
shouldn't we follow the same convention than in travis and put 1.12 first?

Reply:
We could, but the appveyor tests are pretty short. Another PR.